about.html

```diff
@@ -0,0 +1,171 @@
+            <div class="about-page2-container">
+                <div class="about-page2-1">
+                    <div class="about-page2-1-1">
+                        <p class="about-page2-1-1-p">Creative Commons 
+                            Global Summit 2015 Logo</p>
+                        <img src="img/title_underline (2).png" alt="" class="underline">
+                    </div>
+                    <p class="about-page2-2-p">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.</p>
+                </div>
+           
+                <img src="img/about_logo 1.png" alt="">
+                  
+            </div>
+
+        </section>
+
+        <section class="about-page3">
+            <div class="about-page3-part1">
+                <div class="about-page3-part1-up">
+                    <p class="about-page3-part1-text1">See the past CC Global Summits</p>
+                    <img src="img/title_underline (3).png" alt="" class="underline">
+                </div>
+                <p class="about-page3-part1-down">Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+
+            </div>
+            <div class="page3-images">
+                <img src="img/about_2011 1.png" alt="">
+                <img src="img/about_2013 1.png" alt="">
+
+            </div>
+
+        </section>
+        <section class="about-page4-about page4-hide">
+            <div class="page4-container">
+                <div class="second-main"> 
+                    <p class="second-page-main">Partner</p>
+                    <img src="img/title_underline.png" alt="" class="underline">
+                </div>
+            </div>
+            <div class="page4-container1">
+                <img src="img/mozilla 1.png" alt="">
+                <img src="img/googlekorea 1.png" alt="">
+                <img src="img/naver 1.png" alt="">
+            </div>
+            <div class="page4-container2">
+                <img src="img/daumkakaologo 1.png" alt="">
+                <img src="img/airbnb 1.png" alt="">
+
+            </div>
+
+
+        </section>
+
+        <div class="end-about end-hide">
+            <div class="end1">
+                <img src="img/cck-logo1 1.png" alt="" class="end-img">
+                <p class="end-p">2015 Creative Commons Korea. Some Rights Reserved.</p>
+            </div>
+
+        </div>
+        <div class="end end-about2">
+            <div class="end1">
+                <img src="img/cck-logo1 1 (3).png" alt="" class="end-img">
+                <div class="end-pp">
+                    <p class="end-ppp">2015 Creative Commons Korea. Some Rights Reserved.</p>
+                    <p class="end-ppp">달리 정하지 않은 웹사이트의 모든 내용은 크리에이티브 커먼즈 저작자표시 2.0 대한민국 라이선스에 따라 이용할 수 있습니다.
+                        자세한 이용기능 내용과 허락조건은 http://creativecommons.org/licences/by/2.0/kr 를 보세요.</p>
+
+                </div>
+               
+            </div>
```

Comment:
- [ ] You've made excellent progress on your project! In the current implementation, the footer section is using a `<div>` tag. To adhere to semantic HTML best practices, I recommend replacing this `<div>` tag with a `<footer>` tag. Updating the tag to `<footer>` will improve the semantic structure of your code and enhance accessibility. Your attention to these details will contribute to a more well-organized and comprehensible project. Keep up the fantastic work! 🌟📝🚀